### DISCUSSION TITLE
MaxCoreStuckTime

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [azerothcore/azerothcore-wotlk#3988] [General]
I was wondering if anyone has used to run AC with `MaxCoreStuckTime` enabled in `worldserver.conf` and what your feedback is.

It looks like useful to prevent the server to freeze forever

![image](https://user-images.githubusercontent.com/75517/102017258-1bf60a80-3d66-11eb-8c60-51aca4a3e9a5.png)

**Comment by pangolp** [comment]
The truth is that I'm not one to look at the worldserver.conf options. I usually use them by default. I know that I have to check well, because there are many interesting options but for the moment, I haven't done it.
